frontend/src/models.ts

```diff
@@ -177,12 +177,19 @@ export type ExternalConnectionCollection = {
   caldav?: ExternalConnection[];
 };
 
-// The type `Availability` will be used later if we provide custom availabilities
-// in addition to general availability too
 export type Availability = {
   id?: number;
+  schedule_id?: number;
+  day_of_week: number;
+  start_time: string;
+  end_time: string;
+  slot_duration?: number;
 };
 
+export type AvailabilitySet = {
+  [key:number]: Availability[];
+}
```

Comment:
I already prepared this for the next step, when we have multiple availability entries per day.